zstacklib/zstacklib/utils/shell.py

```
@@ -46,8 +46,9 @@ def __init__(self, cmd, workdir=None, pipe=True):
         self.stderr = None
         self.return_code = None
 
-    def raise_error(self):
-        err = []
+    def raise_error(self, err=()):
+        if err is ():
```

Ruff

Use `==` to compare constant literals

Replace `is` with `==`

(F632)